### PR TITLE
Leap 15.3: Add JeOS-efi and JeOS-raspberrypi for aarch64

### PR DESCRIPTION
**Pull request by ggardet**

### CODE REVIEW

File: xml/obs/openSUSE:Leap:15.3:Images.xml [hunk 1]
```diff
@@ -8,6 +8,12 @@
         <!-- Due to there being multiple archs, it omits "head -n1". So don't match .sha256(.asc). -->
         <hdd filemask=".*kvm-and-xen.*\.qcow2$"/>
     </flavor>
+    <flavor name="JeOS-for-AArch64" folder="JeOS:JeOS-efi.aarch64">
```
Comment (Vogtinator) [inline xml/obs/openSUSE:Leap:15.3:Images.xml:11]:
```
rsync: change_dir "/openSUSE:Leap:15.3:Images:ToTest/images/JeOS:JeOS-efi.aarch64" (in openqa) failed: No such file or directory (2)
```

```suggestion
    <flavor name="JeOS-for-AArch64" folder="*/JeOS:JeOS-efi.aarch64">
```

File: xml/obs/openSUSE:Leap:15.3:Images.xml [hunk 2]
```diff
@@ -8,6 +8,12 @@
...
+        <hdd filemask=".*ARM-JeOS-efi.*\.raw\.xz"/>
+    </flavor>
+    <flavor name="JeOS-for-RPi" folder="JeOS:JeOS-raspberrypi.aarch64">
```
Comment (Vogtinator) [inline xml/obs/openSUSE:Leap:15.3:Images.xml:14]:
```suggestion
    <flavor name="JeOS-for-RPi" folder="*/JeOS:JeOS-raspberrypi.aarch64">
```

**Review by Vogtinator — CHANGES_REQUESTED** [review]

File: xml/obs/openSUSE:Leap:15.3:Images.xml (hunk 1 above)
Comment (ggardet) [reply xml/obs/openSUSE:Leap:15.3:Images.xml]: I thought I did it already. Thanks!

File: xml/obs/openSUSE:Leap:15.3:Images.xml (hunk 2 above)
Comment (ggardet) [reply xml/obs/openSUSE:Leap:15.3:Images.xml]: ack

**Review by Vogtinator — APPROVED** [review]

File: xml/obs/openSUSE:Leap:15.3:Images.xml [hunk 3]
```diff
@@ -8,6 +8,12 @@
         <!-- Due to there being multiple archs, it omits "head -n1". So don't match .sha256(.asc). -->
         <hdd filemask=".*kvm-and-xen.*\.qcow2$"/>
     </flavor>
+    <flavor name="JeOS-for-AArch64" folder="*/JeOS:JeOS-efi.aarch64">
+        <hdd filemask=".*ARM-JeOS-efi.*\.raw\.xz"/>
```
Comment (Vogtinator) [inline xml/obs/openSUSE:Leap:15.3:Images.xml:12]: I missed this, it also picked up checksums:

```suggestion
        <hdd filemask=".*ARM-JeOS-efi.*\.raw\.xz$"/>
```

**Review by Vogtinator — CHANGES_REQUESTED** [review]

File: xml/obs/openSUSE:Leap:15.3:Images.xml (hunk 3 above)
Comment (ggardet) [reply xml/obs/openSUSE:Leap:15.3:Images.xml]: Fixed!

**Review by Vogtinator — APPROVED** [review]

**Review by andrii-suse — APPROVED** [review]

**Pull request merged**